ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml

```diff
@@ -48,12 +48,12 @@ Popup {
             color: Style.current.dropShadow
         }
     }
-    x: applicationWindow.width - activityCenter.width - Style.current.halfPadding
+    x: Global.applicationWindow.width - activityCenter.width - Style.current.halfPadding
     onOpened: {
-        popupOpened = true
+        Global.popupOpened = true
```

Comment:
this seems tricky, is this to know if any popup is launched any where?

Can we then not use the instance created in the AppMain.qml when calling openPopup() to connect to th onOpened and inclosed Signals and set it there? Would be become more generic then?

Reply:
Actually I am not really sure what is/was the intention of this property, it is used only in `CompactMessageView.qml` in line 647, enabled property and it's actually set in both `ActivityCenterPopup.qml` and `ModalPopup.qml`. @PascalPrecht would you have any insights here?

Reply:
Hey friends, sorry for the late reply. Catching up on PR reviews...

So doing a quick search for `popupOpened` in the code base, we can see in the `CompactMessageView` that it's used as part of an expression to determine whether or not the hover handler for the message should be enabled.

I believe this was done because the UI was still signalling hover interaction in messages, even when a popup was overlaying the messages.
In other words, it's as if the cursor bleeds through the popup, causing the hover handler of messages to activate.

This is a hack and I assume this can be removed if we figure out why the popup doesn't prevent the underlying messages to receive mouse events.

Reply:
yes, that's normal. what it's used to be done is to have a dummy disabled `MouseArea` inside the `ModalOverlay` to prevent background from consuming click events. Could you/is there already any issue for this?

Reply:
This is the original issue https://github.com/status-im/status-desktop/issues/2187 and here's the PR that "fixed" this https://github.com/status-im/status-desktop/pull/2190

Reply:
I see. However this solves the issue only in the `HoverHandler` in `CompactMessage.qml`, should we work on a solution that's general to the Popups? That said, when a popup opens make sure that no events are consumed in the background?

Reply:
Oh yea absolutely. I rather see something more scalable and robust.
Would you mind creating an issue for this accordingly? Seems like you have a concrete idea of what that could look like.

Reply:
Late reply, but I was the one that added that `popupOpened` property and Pascal got it right on the nose. It was indeed because there was a weird issue with the HoverHandler of CompactMessages that were getting interacted with when a popup was opened.
Feel free to remove `popupOpened` if you find a solution that works and is not as much of a hack.